$OpenBSD: patch-src_startup_c,v 1.6 2015/12/18 15:53:09 dcoppa Exp $

On OpenBSD, time_t is long long.

Turn "char *" into "const char *" for all command parser functions.

--- src/startup.c.orig	Wed Sep 30 08:55:10 2015
+++ src/startup.c	Fri Dec 18 13:58:58 2015
@@ -103,8 +103,8 @@ static int _prune_startup_sequences(void) {
  */
 void startup_sequence_delete(struct Startup_Sequence *sequence) {
     assert(sequence != NULL);
-    DLOG("Deleting startup sequence %s, delete_at = %ld, current_time = %ld\n",
-         sequence->id, sequence->delete_at, time(NULL));
+    DLOG("Deleting startup sequence %s, delete_at = %lld, current_time = %lld\n",
+         sequence->id, (long long)sequence->delete_at, (long long)time(NULL));
 
     /* Unref the context, will be free()d */
     sn_launcher_context_unref(sequence->context);
@@ -239,8 +239,8 @@ void startup_monitor_event(SnMonitorEvent *event, void
             /* Mark the given sequence for deletion in 30 seconds. */
             time_t current_time = time(NULL);
             sequence->delete_at = current_time + 30;
-            DLOG("Will delete startup sequence %s at timestamp %ld\n",
-                 sequence->id, sequence->delete_at);
+            DLOG("Will delete startup sequence %s at timestamp %lld\n",
+                 sequence->id, (long long)sequence->delete_at);
 
             if (_prune_startup_sequences() == 0) {
                 DLOG("No more startup sequences running, changing root window cursor to default pointer.\n");
@@ -261,7 +261,7 @@ void startup_monitor_event(SnMonitorEvent *event, void
  * Renames workspaces that are mentioned in the startup sequences.
  *
  */
-void startup_sequence_rename_workspace(char *old_name, char *new_name) {
+void startup_sequence_rename_workspace(const char *old_name, const char *new_name) {
     struct Startup_Sequence *current;
     TAILQ_FOREACH(current, &startup_sequences, sequences) {
         if (strcmp(current->workspace, old_name) != 0)
